src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 0;
+
+  return (callFirst) => {
+    callCounter++;
+
+    return (callSecond) => {
+      if (callCounter > 3 && callCounter % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return callFirst + callSecond;
+      }
+    };
```

Comment:
```suggestion
      if (callCounter > 3 && callCounter % 2 === 0) {
        return 'Bzzz... Error!';
      }
      return callFirst + callSecond;
    };
```
try to avoid `if else` construction